tracing-appender/src/rolling.rs

```diff
@@ -209,12 +209,19 @@ impl RollingFileAppender {
         })
     }
 
+    #[cfg(test)]
+    #[inline]
+    fn now(&self) -> OffsetDateTime {
+        (self.now)()
+    }
+
+    #[cfg(not(test))]
     #[inline]
     fn now(&self) -> OffsetDateTime {
-        #[cfg(test)]
-        return (self.now)();
+        #[cfg(feature = "local-time")]
+        return OffsetDateTime::now_local().unwrap_or_else(|_| OffsetDateTime::now_utc());
```

Comment:
Not sure about this error handling though, suggestions welcome!

Reply:
IMO we should panic here. Otherwise we would failover "silently" which isn't so nice here.